www/js/diary/diaryHelper.ts

```diff
@@ -144,9 +148,12 @@ export function getFormattedDateAbbr(beginFmtTime: string, endFmtTime?: string)
  */
 export function getFormattedTimeRange(beginFmtTime: string, endFmtTime: string) {
   if (!beginFmtTime || !endFmtTime) return;
-  const beginMoment = moment.parseZone(beginFmtTime);
-  const endMoment = moment.parseZone(endFmtTime);
-  return endMoment.to(beginMoment, true);
+  const beginTime = DateTime.fromISO(beginFmtTime, { setZone: true });
+  const endTime = DateTime.fromISO(endFmtTime, { setZone: true });
+  const range = endTime.diff(beginTime, ['hours']);
+  const roundedHours = Math.round(range.as('hours')); // Round up or down to nearest hour
+  const formattedRange = `${roundedHours} hour${roundedHours !== 1 ? 's' : ''}`;
+  return formattedRange;
```

Comment:
I'm guessing Luxon doesn't support the same kind of textual descriptions for time ranges, so you tried to replicate it like this.

Unfortunately, this will only work for English and only with hours as the unit.
We need a more robust solution that works across languages and on varying scales of time (e.g. "27 minutes", "3 hours", "1 day")

Reply:
Other people have noted that this is missing from Luxon: https://stackoverflow.com/questions/63526184/luxon-interval-human-readable

They suggest an external package [`humanize-duration`](https://www.npmjs.com/package/humanize-duration). I checked and it **does** support Lao.

Reply:
It seems like Luxon `Duration`s do have a `toHuman()` function, but it doesn't do what people expect it to do https://github.com/moment/luxon/issues/1134
Can you look into whether `toHuman()` would be good enough for our purposes? Otherwise, I think we should add the `humanize-duration` package